### ISSUE TITLE
Document HTTP queries for lib microservice

### ISSUE THREAD

**Issue by prasadtalasila** [INTO-CPS-Association/DTaaS#104]
The lib microservice uses graphql API. The graphql querires can be sent as HTTP post queries over the same API connection.

There are two graphql queries for lib microservice. The equivalent HTTP queries need to be documented.

Use Postman for checking HTTP queries.

**Comment by lindot11** [comment]
I have documented the HTTP queries for these docs and made a pull request
- docs/admin/servers/lib/LIB-MS.md
- servers/lib/README.md

Not sure if I can do the last one, because my computer doesn't have the capacity to install the DTaaS application and try the HTTP queries as user.
- docs/user/servers/lib/LIB-MS.md

**Comment by lindot11** [comment]
Regarding the json prettifying issue. 
```
{
   "query":"query { listDirectory(path: \"user1\") { repository { tree { blobs { edges { node { name type } } } trees { edges { node { name type } } } } } } }"
}
```

The query value is one graphql query string, since the value in json should be a single line. 
Alternatively I thought of changing the request body like below so it's easier to identify where to change the path variable in the query.
```
{
   "query": "query ($path: String!) { listDirectory(path: $path) { repository { tree { blobs { edges { node { name type } } } trees { edges { node { name type } } } } } } }",
   "variables": {
      "path": "user1"
   }
}
```

**Comment by prasadtalasila** [comment]
fixed in PR #113 and #115
